rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml

```diff
@@ -190,12 +190,12 @@ index a new document on top of an old one:
 ---
 index a new document on top of an old one over bulk:
   - skip:
-      version: " - 8.1.99"
-      reason: indexing on top of another document support added in 8.2
+      version: "- 8.12.99"
```

Comment:
Let's change this to 8.13.99, since my change also updated the id.

Reply:
Reading the yaml docs, we can just add one skip to the setup block of this file and all tests should be skipped. This should avoid confusion with the all the different skip versions in this file:

```
`skip` can also be used at the top level of the file in the `setup` and `teardown` blocks,
so all the tests in a file will be skipped if the condition applies.
A particular test is skipped if any of the skip conditions for the test,
the setup or the teardown apply.
```

I will just keep one skip version here.